### PR TITLE
An

### PR DESCRIPTION
**Pull request by tnngoan**

### CODE REVIEW

File: components/list.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import SelectBox from "./slectBox";
```
Comment (KevinColemanInc) [inline components/list.js:2]: looks like you had a spelling error here

File: .env [hunk 2]
```diff
@@ -1,7 +0,0 @@
-CLIENT_ID_LINKEDIN: 86t9ro4djktxl5
-CLIENT_SECRET_LINKEDIN: 3KBxNpF1xj7pxNnJ
-
-CLIENT_ID_GITHUB: 17fbb6c1cd49eb981c43
-CLIENT_SECRET_GITHUB: 26dde7e190c02bb5a73c1aad4e3687853787b1ca
```
Comment (KevinColemanInc) [inline .env]: this is still in the git repo

Comment (KevinColemanInc) [reply .env]: https://docs.github.com/en/github/managing-large-files/removing-files-from-a-repositorys-history

File: components/list.js (hunk 1 above)
Comment (tnngoan) [reply components/list.js:2]: I don't see it anymore, I guess you fixed it

File: .env (hunk 2 above)
Comment (tnngoan) [reply .env]: when I git rm --cached .env, it couldn't find the file

Comment (KevinColemanInc) [reply .env]: cuz you deleted it here. create a new .env file and add it. I think that should fix the issue.

**Pull request closed without merging**